lib/galaxy/tools/__init__.py

```diff
@@ -2541,7 +2541,7 @@ def iterate(group_inputs, inputs, state_inputs, other_values=None):
             other_values = ExpressionContext( state_inputs, other_values )
             for input_index, input in enumerate( inputs.itervalues() ):
                 # create model dictionary
-                tool_dict = input.to_dict(trans)
+                tool_dict = input.to_dict(trans, other_values=other_values)
```

Comment:
@guerler Yes, but you'll also need datatype support and metadata for the VTK dataset.  Do you want me to just send you a tarsal via email or do you want the tarsal of entire tool, datatype support and dataset uploaded here?